lib/resources/example_json/r4_examples_location.rb

```diff
@@ -0,0 +1,189 @@
+# frozen_string_literal: true
+
+module Cerner
+  module Resources
+    R4_LOCATION_BUNDLE ||= {
+      "resourceType": 'Bundle',
+      "id": '902e1b49-b9c4-4786-b90b-c9612d478f2a',
+      "type": 'searchset',
+      "link": [
+        {
+          "relation": 'self',
+          "url": 'https://fhir-open.devcerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Location?_id=21250409'
+        }
+      ],
+      "entry": [
+        {
+          "fullUrl": 'https://fhir-open.devcerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Location/21250409',
+          "resource": {
+            "resourceType": 'Location',
+            "id": '21250409',
+            "meta": {
+              "versionId": '6',
+              "lastUpdated": '2018-06-12T18:45:00Z'
+            },
+            "text": {
+              "status": 'generated',
+              "div": '<div xmlns="http://www.w3.org/1999/xhtml"><p><b>Location</b></p><p><b>Name</b>: BE 1 North</p><p><b>Alias</b>: BE 1N</p><p><b>Managing Organization</b>: Baseline East Medical Center</p><p><b>Part Of</b>: 21250403</p><p><b>Physical Type</b>: Facility(s)</p><p><b>Status</b>: Active</p></div>' # rubocop:disable Layout/LineLength
+            },
+            "extension": [
+              {
+                "extension": [
+                  {
+                    "id": '20356268',
+                    "valueString": 'Core Measure Locations',
+                    "url": 'custom-attribute-name'
+                  },
+                  {
+                    "valueCodeableConcept": {
+                      "coding": [
+                        {
+                          "system": 'https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/4678015',
```

Comment:
Can we update the tenant to - ec2458f2-1e24-41c8-b71b-0e701af7583d. Also below.

Reply:
Updated here: https://github.com/cerner/fhir.cerner.com/pull/520/commits/2a8f6925fc92f6a6471674fb55164a6fc0807058#diff-77695d88dd05c72e79566d7e2e7deb1f8c620cb7b0d1a871f7826998c3c5a54aR28